upup/pkg/fi/nodeup/command.go

```diff
@@ -760,20 +762,34 @@ func getNodeConfigFromServer(ctx context.Context, bootConfig *nodeup.BootConfig,
 		Resolver:      resolver,
 	}
 
-	u, err := url.Parse(bootConfig.ConfigServer.Server)
-	if err != nil {
-		return nil, fmt.Errorf("unable to parse configuration server url %q: %w", bootConfig.ConfigServer.Server, err)
-	}
-	client.BaseURL = *u
-	client.CAs = []byte(bootConfig.ConfigServer.CACertificates)
+	return loopThroughConfigservers(ctx, client, bootConfig.ConfigServers)
+}
 
-	request := nodeup.BootstrapRequest{
-		APIVersion:        nodeup.BootstrapAPIVersion,
-		IncludeNodeConfig: true,
+// loopThroughConfigservers returns BootstrapResponse we need to loop through all configservers because only one of them is active. Normally loadbalancer healthcheck will take care of detecting the active one.
+// However, like in case of OpenStack there are 3 config servers and no loadbalancer in front of which means that we need loop through all of them to see which one is the active.
+func loopThroughConfigservers(ctx context.Context, client *kopscontrollerclient.Client, configServers []*nodeup.ConfigServerOptions) (*nodeup.BootstrapResponse, error) {
+	var err error
+	for _, configServer := range configServers {
+		u, loopErr := url.Parse(configServer.Server)
+		if loopErr != nil {
+			return nil, fmt.Errorf("unable to parse configuration server url %q: %w", configServer.Server, loopErr)
```

Comment:
Would be simpler to just use `"go.uber.org/multierr"`:
```suggestion
	var merr error
	for _, configServer := range configServers {
		u, loopErr := url.Parse(configServer.Server)
		if loopErr != nil {
			merr = multierr.Append(merr, fmt.Errrorf("unable to parse configuration server url %q: %w", configServer.Server, loopErr))
```